Sources/KlaviyoForms/InAppForms/Models/InAppFormsConfig.swift

```diff
@@ -5,13 +5,23 @@
 //  Created by Isobelle Lim on 5/7/25.
 //
 import Foundation
+import OSLog
 
 /// Configuration for in-app forms
 public struct InAppFormsConfig {
+    /// - Parameter sessionTimeoutDuration: Duration (in seconds) of user inactivity after which the form session is terminated.
+    ///   Defaults to 1 Hour, must be non-negative. Use 0 to timeout as soon as the app is backgrounded.
+    ///   To disable session timeout altogether, use TimeInterval.infinity.
```

Comment:
oh actually just out of curiosity, do the places where we _use_ sessionTimeoutDuration need any update for when it is set to infinity, or will it just work as written.

Reply:
oh wait using time interval is way better than using Long, maybe I should update to kotlin's Duration type before we release